internal/juju/applications.go

```diff
@@ -330,7 +340,8 @@ func (c applicationsClient) CreateApplication(input *CreateApplicationInput) (*C
 	return &CreateApplicationResponse{
 		AppName:  appName,
 		Revision: *origin.Revision,
-		Series:   seriesToUse,
+		Series:   resultOrigin.Series,
+		Base:     fmt.Sprintf("%s@%s", resultOrigin.Base.Name, resultOrigin.Base.Channel.Track),
```

Comment:
These info in the `CreateApplicationResponse` might be redundant. We seem to be using only the `AppName` from this.

Reply:
Point